Multi-cloud documentation (#18872)
airbytehq/airbyte: docs/cloud/managing-airbyte-cloud.md

| Multiple             | Workspaces are independent of each other, so you can use a different payment method card for each workspace (for example,  different credit cards per department in your organization). | You can use the same payment method for different workspaces, but each workspace is billed separately. Managing billing for each workspace can become complicated if you have many workspaces. |

### Switch between multiple workspaces

To switch between workspaces:

1. On the [Airbyte Cloud](http://cloud.airbyte.io) dashboard, click the current workspace name under the Airbyte logo in the navigation bar.

2. Click **View all workspaces**.

3. Click the name of the workspace you want to switch to.

### Choose your default data residency
Default data residency allows you to choose where your data is processed. When you set the default data residency, it applies that data residency to all new connections, but it will not affect existing connections. 

> For individual connections, you can choose a data residency that is different from the default. You can do this in the [connection settings](#choose-the-data-residency-for-a-connection) or when you create a [new connection](https://docs.airbyte.com/cloud/getting-started-with-airbyte-cloud#set-up-a-connection). Your data will not leave the chosen data residency for your connection.

Reviewer: I think this is an area where we might want to talk about the control plane vs. the data plane. Because this setting doesn't necessarily mean where your data is stored, but it's about where your data is processed.

Reviewer 2: that's a good point - we don't highlight the difference between the control/data plane today i.e. while data is processed in the specific region, configuration information is still stored in the US.

this means that users need to be careful to make sure they don't choose a sensitive cursor.
 
Andy, do you think this is worth calling out? Or is it too 'scary' to do so?

Reviewer: I think it's worth mentioning!

Author: I agree that the fact that data is processed in one region and stored in another needs to be mentioned. I'm wondering if I should delete the last line "Your data will not leave the chosen data residency for your connection" and add a note like "Your data is processed in the chosen residency, but some data is still stored in the US." We could also list out types of data (like configuration information) we store in the US if that would help the users.

Any thoughts on that?

Reviewer 2: >  should delete the last line "Your data will not leave the chosen data residency for your connection" and add a note like "Your data is processed in the chosen residency, but some data is still stored in the US." 

I like that. I would like the note to be a markdown note so it jumps out.

One tactical point, we don't have to wordsmith too much in this PR. We can merge this in for now and iterate on subsequent PRs.

Author: @andyjih @davinchia Do you think that a note like this would have enough info about data plane and control plane? Is there any other info you think the users will need to know?  (I'll edit wording after we know what info we want to include):

Your data is processed on a data plane in the chosen residency, but some data, like data associated with the sync mode, cursor, and primary key, is still stored in the US on our control plane.

Reviewer 2: That's pretty good. I would also add: "Please use caution when selecting the cursor and primary key columns."

    The **Connection State** displays. 

## Choose the data residency for a connection
To choose the data residency for an existing connection: 

1. On the [Airbyte Cloud](http://cloud.airbyte.io) dashboard, click **Connections** in the navigation bar and then click the connection that you want to change. 

    The Connection page displays. 

2. Click the **Settings** tab. 

> 3. Click the **Data residency** dropdown and choose the location for your default data residency.

Reviewer 3: Should we add a note here about whether changing a connection's data residency will not affect currently running syncs? @timroes mentioned this when reviewing the UI PR. I don't _think_ currently running syncs will be affected, but I'm not 100% sure. Maybe @davinchia has an idea?

Reviewer 2: Thanks for reminding me Joey - yeah updating the workspace doesn't update existing connections so we should call that out too.

@sophia-wiley one more follow up change!

Author: Okay so just to make sure I understand this correctly -- If you choose a different data residency in the connection settings while a sync is running, it will not affect the currently running sync, but it will apply the new data residency to future syncs for that connection?

Reviewer 2: * Data residency changes will only affect the next sync, and not any currently running sync.
* Changing the workspace default residency only affects newly created connections. Already created connections have to be manually edited.

Does ^ make the behaviour clearer?

:::note 
    

> You can also choose data residency when creating a [new connection](https://docs.airbyte.com/cloud/getting-started-with-airbyte-cloud#set-up-a-connection), or you can set the [default data residency](#choose-your-default-data-residency) for all of your connections. 

Reviewer 3: ```suggestion
You can also choose data residency when creating a [new connection](https://docs.airbyte.com/cloud/getting-started-with-airbyte-cloud#set-up-a-connection), or you can set the [default data residency](#choose-your-default-data-residency) for your workspace.
```

Just a suggestion: this could be interpreted such that changing your default data residency affects all of your connections, which it does not.

Author: Good catch! I'll edit this in another PR
